source/_integrations/google_stackdriver.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Google Stackdriver"
+description: "Setup for Google Stackdriver integration"
+logo: google_stackdriver.png
+ha_category:
+  - History
+ha_release: 0.104
+---
+
+The `google_stackdriver` integration allows you to stream logs to [Google Stackdriver Logging](https://cloud.google.com/logging/). The current [free tier](https://cloud.google.com/free/) of GCP should allow you to log up to 50 GiB.
+
+## First time setup
+
+This assumes you already have a Google Cloud project. If you don't, please create one in the [Google Cloud Console](https://console.cloud.google.com/projectcreate)
+
+You need to create a Service Account key in the [Google Cloud API Console](https://console.cloud.google.com/apis/credentials/serviceaccountkey)
+- Choose a new "New Service Account", give it a name and leave the key type as JSON
+- Select the role: Logs Writer 
+
+This will download the Service Account JSON key to your machine. Do NOT share this with anyone. Place this file in your Home Assistant config folder.
+
+
+## Configuration
+
+Add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+google_stackdriver:
+  key_file: frog32ch_key.json
+  labels:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
OK went with a minimal example. People using Stackdriver already would probably understand how to come up with valid yaml so no extended example needed.